mv_avs_dir.sh

```diff
@@ -52,32 +52,48 @@ usage() {
     output "usage: ${bold}$SCRIPT${normal} <source-directory> <destination-directory>"
 }
 
-rm_if_link(){
-    [ ! -L "$1" ] || rm "$1";
-}
 
-rm_simlink() {
-    cd $1
+## Operates files
+##   - Removes current similinks
+##   - Creates new simlinks
+## @param destination-directory
+## @param delimiter
+##
+op_files() {
+    ##
+    ## Removes simlinks
+    ##   and Creates pair of {simlink, original}
+    ##
+    dir=$1
+    cd "$dir"
+    declare -A array
+
     for i in *
     do
-        rm_if_link "$i"
+        f=$i
+        if  [ -L "$f" ] ; then
+            l=`readlink -f "$f"`
+            l=${l##*/}
+            rm "$f"
+            array[$l]=$f
+        fi
     done
-    cd -
-}
 
-create_simlink() {
-    dir=$1
-    cd "$dir"
+    ##
+    ## Creates simlinks
+    ##
     IFS=$2
 
     for i in *
     do
         org=$i
-        set -- $i
-        source_file="$dir/$org"
-        target_file="$dir/$1"
-        ln -s "$source_file" "$target_file"
-        echo "[CREATED] $source_file > $target_file"
+        val=${array[$org]}
+        if [ -n "$val" ]; then
+            source_file="$dir/$org"
+            target_file="$dir/$val"
+            ln -fs "$source_file" "$target_file"
+            echo "[CREATED] $source_file > $target_file"
+        fi
     done
```

Comment:
L82 - L97 can be removed as I mentioned above.

In summary,
After fixing the whole review comments here, the patch looks like below. 

```diff
diff --git a/mv_avs_dir.sh b/mv_avs_dir.sh
index fbd4ec9..c63ceb4 100755
--- a/mv_avs_dir.sh
+++ b/mv_avs_dir.sh
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
 ## ======================================================================
 ##
 ## LeoFS
@@ -66,35 +66,21 @@ op_files() {
     ##
     dir=$1
     cd "$dir"
-    declare -A array

     for i in *
     do
         f=$i
         if  [ -L "$f" ] ; then
-            l=`readlink -f "$f"`
+            l=`readlink "$f"`
             l=${l##*/}
             rm "$f"
-            array[$l]=$f
-        fi
-    done
-
-    ##
-    ## Creates simlinks
-    ##
-    IFS=$2
-
-    for i in *
-    do
-        org=$i
-        val=${array[$org]}
-        if [ -n "$val" ]; then
-            source_file="$dir/$org"
-            target_file="$dir/$val"
+            source_file="$dir/$l"
+            target_file="$dir/$f"
             ln -fs "$source_file" "$target_file"
             echo "[CREATED] $source_file > $target_file"
         fi
     done
+
     cd -
 }
```